llvm/lib/Target/DirectX/DXILShaderFlags.cpp

```diff
@@ -273,43 +320,6 @@ void ModuleShaderFlags::initialize(Module &M, DXILResourceTypeMap &DRTM,
       // Merge SCCSF with that of F
       FunctionFlags[F].merge(SCCSF);
   }
-
-  // Set DisableOptimizations flag based on the presence of OptimizeNone
```

Comment:
I did not see anything in the description about why we are removing the `DisableOptimizations` block of code.

Reply:
It's not removed; it has been relocated to the new `gatherGlobalModuleFlags` function